src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporter.cs

```diff
@@ -38,24 +40,33 @@ internal class OtlpLogExporter : BaseExporter<LogRecord>
         /// </summary>
         /// <param name="options">Configuration options for the exporter.</param>
         public OtlpLogExporter(OtlpExporterOptions options)
-            : this(options, null)
+            : this(options, new SdkLimitOptions(), null)
```

Comment:
nit: Only cause this is what you did with the trace exporter

```suggestion
            : this(options, new(), null)
```

Reply:
Updated